cookbooks/arcgis-server/recipes/sql_alias.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Cookbook Name:: arcgis-server
+# Recipe:: sql_alias
+#
+# Copyright 2015 Esri
```

Comment:
File created and published this year, should have this year listed for copyright